src/plume-nest/DinariAdapterToken.sol

```diff
@@ -101,6 +98,11 @@ contract DinariAdapterToken is ComponentToken, ReentrancyGuardTransientUpgradeab
 
     // Override Functions
 
+    /// @inheritdoc IComponentToken
```

Comment:
It looks like this is already implemented in `ComponentToken` and can be removed